src/schedule-and-details/introducing-section/index.jsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  injectIntl,
+  intlShape,
+  FormattedMessage,
+} from '@edx/frontend-platform/i18n';
+import { Form, Hyperlink } from '@edx/paragon';
+
+import IntroductionVideo from './introduction-video';
+import CourseCodeEditor from './course-code-editor';
+import CourseCardImage from './course-card-image';
+import messages from './messages';
+
+const IntroducingSection = ({
+  intl,
+  courseId,
+  overview,
+  introVideo,
+  aboutSidebarHtml,
+  shortDescription,
+  aboutPageEditable,
+  sidebarHtmlEnabled,
+  lmsLinkForAboutPage,
+  courseImageAssetPath,
+  shortDescriptionEditable,
+  onChange,
+}) => {
+  const overviewHelpText = (
+    <FormattedMessage
+      id="course-authoring.schedule-section.introducing.course-overview.help-text"
+      defaultMessage="Introductions, prerequisites, FAQs that are used on {hyperlink} (formatted in HTML)"
+      values={{
+        hyperlink: (
+          <Hyperlink
+            destination={lmsLinkForAboutPage}
+            target="_blank"
+            rel="noopener noreferrer"
+            showLaunchIcon={false}
+          >
+            {intl.formatMessage(messages.courseAboutHyperlink)}
+          </Hyperlink>
+        ),
+      }}
+    />
+  );
+
+  const aboutSidebarHelpText = (
+    <FormattedMessage
+      id="course-authoring.schedule-section.introducing.about-sidebar.help-text"
+      defaultMessage="Custom sidebar content for {hyperlink} (formatted in HTML)"
+      values={{
+        hyperlink: (
+          <Hyperlink
+            destination={lmsLinkForAboutPage}
+            target="_blank"
+            rel="noopener noreferrer"
+            showLaunchIcon={false}
+          >
+            {intl.formatMessage(messages.courseAboutHyperlink)}
+          </Hyperlink>
+        ),
+      }}
+    />
+  );
+
+  return (
+    <section className="section-container details-section">
+      {aboutPageEditable && (
+        <header className="section-header">
+          <span className="lead">
+            {intl.formatMessage(messages.introducingTitle)}
+          </span>
+          <span className="x-small text-gray-700">
```

Comment:
Same question for description. Should there be an appropriate paragraph tag here?